.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: android
+android:
+  components:
+  - build-tools-25.0.0
+  - android-26
+  - extra-google-google_play_services
+  - extra-google-m2repository
+  - extra-android-m2repository
+  - addon-google_apis-google-25
```

Comment:
You need to add the licenses under the android section to let Travis download  the Android sdk:
```
android:
 ...
 licenses:
    - 'android-sdk-license-.+'
```

Reply:
Thanks for your comment, I have added it to travis.yml file :)